Commands/Commands/utils.js

```diff
@@ -243,8 +243,8 @@ commands['approve'] = {
                 })
                 return
             } else {
-                if (state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
-                    message.reply('You\'ve already approved this report.').then(delay(config.timeouts.messageDelete)).then((msg) => {
+                if (state[content[0]].deniedUsers.indexOf(message.author.id) > -1 || state[content[0]].approvedUsers.indexOf(message.author.id) > -1) {
```

Comment:
The point of letting you deny the report even tough you approved it was to let you change your mind, this works (somewhat) the same way on Discord Testers too.